server/webapp/WEB-INF/rails/webpack/models/pipeline_configs/spec/job_spec.ts

```diff
@@ -63,4 +65,21 @@ describe("Job model", () => {
     });
   });
 
+  it("should exclude env vars with no name when serializing", () => {
```

Comment:
We shouldn't allow this case for a valid job. Validate on `EnvironmentVariableConfig`.